### DISCUSSION TITLE
Github copilot stopped working

### DISCUSSION THREAD

**Discussion by magick93** [community/community#76718] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I've been a paying user of this product for a while now. A few days it stopped working. 

I've tried:
- disabling all extensions
- re-install co-pilot
- sacrificing goats
- removing the `/home/anton/.cache/Code` cache
- uninstalling and re-installing VScode
- removing ALL other extensions

Nothing has worked. 

# Diagnostics

## Copilot

- Version: 1.138.0
- Build: prod
- Editor: vscode/1.84.2

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Custom Certificates: disabled
- Send Restricted Telemetry: enabled
- Chat: enabled

## Node setup

- Number of root certificates: 140
- Operating system: Linux
- Operating system version: 6.5.0-13-generic
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Proxy auth: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

**Comment by deeptejd** [comment]
It happens to me too but it tends to automatically resolve on its own. Generally that happens when I have a weaker internet connection, but I've noticed it on stronger ones too sometimes. at this point it just seems like a bug

**Reply by magick93** [reply]
My internet is as strong as Thor.
